src/server.cc

```diff
@@ -4,3 +4,17 @@
 // file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
 #include "server.h"
+
+void Server::do_accept()
+{
+  acceptor_.async_accept(socket_,
+      [this](boost::system::error_code ec)
+      {
```

Comment:
Sure. I just used vim's built-in indentation engine. I'll keep an eye out for it now.